api/sample.env

```diff
@@ -156,6 +156,17 @@ TEST_DATABASE_URL=postgresql://postgres@localhost/pix_test
 # default: none
 # SENDINBLUE_ORGANIZATION_INVITATION_TEMPLATE_ID=
 
+# ID of the template used for generating the e-mail when a user want to
+# join a SCO organization.
+#
+# If not present when required, the e-mail will not be sent and an error will
+# be thrown.
+#
+# presence: required only if emailing is enabled and provider is Sendinblue
+# type: Number
+# default: none
+# SENDINBLUE_ORGANIZATION_INVITATION_SCO_TEMPLATE_ID=
```

Comment:
Bien vu le sample.env !
Un peu hors sujet, mais sauf erreur de ma part, elle ne figure pas dans le [Wiki](https://1024pix.atlassian.net/wiki/spaces/DEV/pages/957677602/2019-07-16+Glossaire+des+variables+d+environnement)

Reply:
Merci @octo-topi Je m'étais bien noté la MAJ du Wiki, lors de la MEP de ce ticket :)